client/lib/plans/features-list.js

```diff
@@ -1321,14 +1337,16 @@ export const FEATURES_LIST = {
 		getSlug: () => constants.FEATURE_PRODUCT_SEARCH_V2,
 		getIcon: () => ( getJetpackCROActiveVersion() === 'v2' ? 'search' : null ),
 		getTitle: () =>
-			getJetpackCROActiveVersion() === 'v2'
-				? i18n.translate( 'Jetpack Search {{strong}}{{em}}Up to 100k records{{/em}}{{/strong}}', {
-						components: {
-							em: <em />,
-							strong: <strong />,
-						},
-				  } )
-				: i18n.translate( 'Search: up to 100k records' ),
+			( {
+				v2: i18n.translate( 'Jetpack Search {{strong}}{{em}}Up to 100k records{{/em}}{{/strong}}', {
+					components: {
+						em: <em />,
+						strong: <strong />,
+					},
+				} ),
+				i5: i18n.translate( 'Site Search: up to 100k records' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'Search: up to 100k records' )` *ES Score: 11*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/improve/pricing-page-i5-update-copy)